Python/callable-functions/main.py

```diff
@@ -0,0 +1,147 @@
+# Copyright 2023 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https:#www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import re
+from typing import Any
+
+from firebase_admin import db, initialize_app
+from firebase_functions import https_fn, options
+
+initialize_app()
+
+# [START v2allAdd]
+# [START v2addFunctionTrigger]
+# Adds two numbers to each other.
+@https_fn.on_call(
+    cors=options.CorsOptions(cors_origins="*", cors_methods=["get", "post", "put"])
```

Comment:
I believe CORS is set up automatically for callables - shouldn't need to set this option